src/main/java/com/minecolonies/coremod/entity/ai/citizen/lumberjack/Tree.java

```diff
@@ -121,6 +121,11 @@
      */
     private boolean dynamicTree = false;
 
+    /**
+     * If the tree is a Nether Tree
+     */
+    private boolean netherTree = false;
```

Comment:
I'm pretty sure we write this tree to NBT somewhere too, so we have to write this boolean too.